lib/apiary/command/preview.rb

```diff
@@ -30,11 +30,15 @@ def initialize(opts)
         @options = OpenStruct.new(opts)
         @options.path         ||= 'apiary.apib'
         @options.api_host     ||= 'api.apiary.io'
-        @options.headers      ||= {:accept => 'text/html', :content_type => 'text/plain'}
         @options.port         ||= 8080
         @options.proxy        ||= ENV['http_proxy']
         @options.server       ||= false
-	@options.host	      ||= '127.0.0.1'
+	      @options.host         ||= '127.0.0.1'
+        @options.headers      ||= {
+          :accept => 'text/html',
+          :content_type => 'text/plain',
```

Comment:
We are sending as plain text document. We can't have detection for swagger/api blueprint in ruby.